mobify/platform-scaffold
# [Integration manager] merlins cart

This PR adds the cart page actions to the merlins connector

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1270
 **Linked PRs**: n/a

## Changes
- Move cart commands (remove, update, fetch data etc) to the merlins connector

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart
- view the cart page
- confirm everything works as expected
